doc/api/buffer.markdown

```diff
@@ -351,7 +351,7 @@ for (var i = 0; i < str.length ; i++) {
   buf[i] = str.charCodeAt(i);
 }
 
-console.log(buf);
+console.log(buf.toString());
```

Comment:
The example states it is an ASCII string. I would be explicit and write `buf.toString('ascii')`.